PANDORA/PMHC/PMHC.py

```diff
@@ -258,15 +259,28 @@ def __init__(self, id, peptide, allele_type=[], MHC_class = 'I',
             use_netmhcpan (bool): If True, uses local installation of NetMHCPan to predict the anchors when
                                   anchor positions are not provided. Defaults to False.
             use_templ_seq (bool): If True, if no MHC chain sequences could be retrieved starting from the allele name,
-                                    it will use the best template MHC sequences for the modelling.
-        '''
+                                   it will use the best template MHC sequences for the modelling.
+            output_dir: (string) Path to output directory. Defaults to os.getcwd().
```

Comment:
please change "Defaults to os.getcwd()." to "Defaults to current working directory"